agent/php_execute.c

```diff
@@ -491,8 +491,8 @@ static nr_library_table_t libraries[] = {
      * /usr/local/bin. While BaseTestRunner isn't the very first file to load,
```

Comment:
Should the comment be updated since we don't use BaseTestRunner anymore?

Reply:
Good catch! Comment updated in bd240c915ee0bce31a3c0802eaa4af287cd10fff.